### DISCUSSION TITLE
Typescript-eslint v8 - `projectService`

### DISCUSSION THREAD

**Discussion by AndreaPontrandolfo** [AndreaPontrandolfo/sheriff#160] [Ideas]
Discussion about potentially removing `project: true` and implementing `projectService: true`.
Originated from: https://github.com/AndreaPontrandolfo/sheriff/pull/156#issuecomment-2221486899

**Comment by lishaduck** [comment]
Pros:
- Faster
- Less configuration
- Doesn't need tsconfig.eslint.json
Cons:
- Doesn't support tsconfig.eslint.json (for, i.e., web projects), or tsconfig.sw.json (for service workers)

**Comment by AndreaPontrandolfo** [comment]
Closing in favor of https://github.com/AndreaPontrandolfo/sheriff/issues/177
